rsmtool/analyzer.py

```diff
@@ -613,56 +606,67 @@ def metrics_helper(human_scores,
-            `use_diff_std_means=True`, this must be used with `population_human_score_mn`.
-            Otherwise, it is ignored.
-            Defaults to None.
+            Reference standard deviation for human scores.
+            If ``smd_method`` is set to "williamson", this is used to
+            compute SMD and should be the standard deviation for the whole
+            population. If ``use_diff_std_means`` is ``True``, this must be
+            used with ``population_human_score_mn``. Otherwise, it is ignored.
+            Defaults to ``None``.
         population_system_score_sd : float, optional
-            Reference standard deviation for system scores. If `smd_method='williamson'`, this is
-            used to compute SMD and should be the standard deviation for the whole population.If
-            `use_diff_std_means=True`, this must be used with `population_system_score_mn`.
-            Otherwise, it is ignored.
-            Defaults to None.
+            Reference standard deviation for system scores. If
+            ``smd_method`` is set to "williamson", this is used to compute
+            SMD and should be the standard deviation for the whole population.
+            If ``use_diff_std_means`` is ``True``, this must be used with
+            ``population_system_score_mn``. Otherwise, it is ignored.
+            Defaults to ``None``.
         population_human_score_mn : float, optional
-            Reference mean for human scores. If `use_diff_std_means=True`, this must be used with
-            `population_human_score_sd`. Otherwise, it is ignored.
-            Defaults to None.
+            Reference mean for human scores. If ``use_diff_std_means`` is
+            ``True``, this must be used with ``population_human_score_sd``.
+            Otherwise, it is ignored.
+            Defaults to ``None``.
         population_system_score_mn : float, optional
-            Reference mean for system scores. If  `use_diff_std_means=True`, this must be used with
-            `population_system_score_sd`. Otherwise, it is ignored.
-            Defaults to None.
-        smd_method : {'williamson', 'johnson', pooled', 'unpooled'}, optional
-            The SMD method to use, only used if `use_diff_std_means=False`.
-            All methods have the same numerator mean(`y_pred`) - mean(`y_true_observed`)
-            and the following denominators :
-
-            - `williamson`: pooled population standard deviation of `y_true_observed` and `y_pred`
-            - `johnson`: population standard deviation of `y_true_observed`.
-            - `pooled`: pooled standard deviation of `y_true_observed` and `y_pred` for this group.
-            - `unpooled`: standard deviation of `y_true_observed` for this group.
-
-            Defaults to 'unpooled'.
+            Reference mean for system scores. If  ``use_diff_std_means`` is
+            ``True``, this must be used with ``population_system_score_sd``.
+            Otherwise, it is ignored.
```

Comment:
I think you mean "must be the mean ..." instead of "must be the standard deviation ..." but you can double check when you review again after my commits.